qingstor/sdk/client/upload_client.py

```diff
@@ -0,0 +1,82 @@
+from qingstor.sdk.service.qingstor import QingStor
+from file_chunk import FileChunk
+from ..constants import HTTP_CREATED,HTTP_OK,HTTP_BAD_REQUEST
+from ..error import PartTooSmallError,MaxPartsExceededError
+import logging
+import os
+class UploadClient:
```

Comment:
Should have an empty line here.
Use [yapf](https://github.com/google/yapf) for code format.